Podfile

```diff
@@ -9,6 +9,11 @@ target 'PushNotificationDemo' do
 
   target 'PushNotificationDemoUITests' do
     # Pods for testing
-    pod 'SBTUITestTunnelHost', :git => 'https://github.com/Subito-it/SBTUITestTunnelHost.git'
```

Comment:
I used this to pin to the master branch to always get the latest commit and avoid to rely on the central cocoapods spec.